app/helpers/application_helper.rb

```diff
@@ -384,8 +384,8 @@ def gravatar_image(email)
   end
 
   def gravatar_html_options
-    { :onerror => "this.src='#{path_to_image('user.jpg')}'",
-      :alt     => _('Change your avatar at gravatar.com') }
+    { :onerror => "$(this).addClass('fa fa-user')",
+      :alt     => '' }
```

Comment:
why drop the comment? if gravatar is enabled users should know how to use it?

Reply:
alternative text renders when error occurred while loading the image.
Now we're using icon instead, so we can use `title`attribute instead.
anyway, that text comment wasn't be visible for uses before

Reply:
ACK - how hard would it be to write a test for this?